docs/pre-commit.md

```diff
@@ -65,22 +65,25 @@ to use for the pre-commit check.
 
 In the above section, the `rev` value was set to `main`:
 
-```Yaml
```

Comment:
I think I see where this is going.  Having the complete yaml section in these samples does make sense to me.   Can you add the `repos:` line as in the comment above?